bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2022.html

```diff
@@ -0,0 +1,470 @@
+    <p>In the United States, we testified at a US Congressional hearing on privacy; we published a <a href="https://blog.mozilla.org/en/mozilla/americans-deserve-federal-privacy-protections-and-greater-transparency-into-hidden-harms-online/">blog post</a> detailing the testimony. In close coordination with allies, we also engaged on ad transparency and researcher access with the Federal Trade Commission (FTC) and Congress. In the United Kingdom, we continued our work on privacy preserving advertising and mitigating online tracking via <a href="https://blog.mozilla.org/netpolicy/2022/04/12/competition-should-not-be-weaponized-to-hobble-privacy-protections-on-the-open-web/">submissions</a> and conversations with the Competition and Markets Authority (CMA) and the Information Commissioner's Office (ICO). In India, we were <a href="https://www.ft.com/content/7b6f91c3-8324-467b-953a-d5325a5aacd3">quoted</a> in the Financial Times on contact tracing apps and the risks they pose to civil liberties, based on our previous <a href="https://www.aninews.in/news/national/general-news/decision-to-open-source-aarogya-setu-app-will-help-improve-user-trust-security-mozilla20200527153954/">work</a> on India’s contact tracing app and its open sourcing. We also <a href="https://portswigger.net/daily-swig/indian-vpn-providers-resist-incoming-data-logging-law">engaged</a> on India’s CERT-In regulations surrounding vastly expanded data retention requirements for VPN service providers. In the Philippines, our <a href="https://mb.com.ph/2022/04/11/mozilla-bucks-philippine-sim-card-registration-law/">outreach</a> (working closely with the community) played a role in the <a href="https://www.philstar.com/headlines/2022/04/15/2174584/duterte-rejects-bill-requiring-sim-card-social-media-account-registration">vetoing</a> of the SIM Card Registration Act which mandated that ‘social media platforms’ (which were very vaguely defined) verified the identity of real world users. In the African continent, we also ramped up our Lean Data Practices <a href="https://blog.mozilla.org/netpolicy/2022/01/26/lean-data-practice-journey/">work</a> by taking our online <a href="https://www.udemy.com/course/lean-data-practices/">course</a> to new audiences from the startup ecosystem.</p>
+
+    <h3>Voluntary Threat Indicators & Data Disclosures</h3>
+    <table class="mzp-u-data-table">
+      <thead>
+        <tr>
+          <th scope="col">Type of Disclosure</th>
+          <th scope="col">Number of Disclosures</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th scope="row"><a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-threat-indicator' }}">Cybersecurity Threat Indicator</a>
+          </th>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">
+            Other <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-specific-user' }}">Specific User</a> Data Disclosure
```

Comment:
**question (curious):** what's the `dfn` namespace stand for?

Reply:
I can only assume it's short for "definition"?

Reply:
sounds very reasonable, thanks!